R/zcurve.R

```diff
@@ -0,0 +1,31 @@
+#' Create a curve comparing observed Z-scores to the WHO standard.
+#'
+#' @param zscore a numeric vector containing computed zscores
+#' @return a ggplot2 object
```

Comment:
Maybe add a line that users can extend that object.

Reply:
Done